### ISSUE TITLE
Python: Write punctuation markers with Stream.print() to stdout

### ISSUE THREAD

**Issue by markheger** [IBMStreams/streamsx.topology#2516]
Stream.print() has a new parameter to print the punctuations to stdout.

```
topo = Topology()
s = topo.source([1,2,3,4])
s = s.punctor(lambda t : 2 < t)
s.print(write_punctuations=True)
```

**Comment by markheger** [comment]
merged to IBMStreams:feature/punct

**Comment by markheger** [comment]
merged to develop branch for 1.16.0a
